helm/ffc-pay-data-hub/values.yaml

```diff
@@ -37,6 +37,9 @@ container:
   redisPassword: redis
   redisPort: 6379
   redisPartition: ffc-pay-data-hub
+  azureStorageConnectionString: dummy
```

Comment:
dummy?

Reply:
Can't put the real string so making it clear it's not real and needs a value.

Dummy used to be consistent with other repos